src/morphology/english/determineStem.js

```diff
@@ -13,17 +13,26 @@ import { flatten } from "lodash-es";
  *
  * @returns {string} The shortest of the alphabetically ordered strings from the input array.
  */
-export function getFirstAndShortest( array ) {
-	array = uniq( flatten( array ) );
-
-	// Sort alphabetically.
-	array.sort();
+export function findShortestAndAlphabeticallyFirst( array ) {
+	if ( array.length === 0 ) {
```

Comment:
This is not needed with that implementation, here is why:
`[].pop()` returns `undefined`.
`[].forEach()` just doesn't loop.
The return in the end will return the `undefined` result. Which is the same thing this if statement is doing.

Reply:
Fair point!